sdk/tables/azure-data-tables/azure/data/tables/_serialize.py

```diff
@@ -185,6 +185,14 @@ def _to_entity_none(value):  # pylint: disable=unused-argument
     EdmType.INT32: _to_entity_int32,
     EdmType.INT64: _to_entity_int64,
     EdmType.STRING: _to_entity_str,
+    "EdmType.BINARY": _to_entity_binary,
+    "EdmType.BOOLEAN": _to_entity_bool,
+    "EdmType.DATETIME": _to_entity_datetime,
+    "EdmType.DOUBLE": _to_entity_float,
+    "EdmType.GUID": _to_entity_guid,
+    "EdmType.INT32": _to_entity_int32,
+    "EdmType.INT64": _to_entity_int64,
+    "EdmType.STRING": _to_entity_str,
```

Comment:
You shouldn't need to add these here - and the values should be lowercase, e.g. 'Edm.Int64':
https://docs.microsoft.com/en-us/rest/api/storageservices/understanding-the-table-service-data-model#property-types
Because the enum inherits froms string - the dict lookup on line 237 _should_ just work....